### PR TITLE
Improve subscription payload tests and structure

### PR DESCRIPTION
**Pull request by jakubkuc**
I want to merge this change because generating payloads from subscription needs to be more throughly tested.
This PR changes structure of tests for subscriptions so in the future we can use its fragments and queries throughout whole project.

There are still payloads that needs to be expanded (checkout, order, product) but we decided to add them later in separate prs.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6516 Improve tests for async webhooks](https://app.clickup.com/t/2549495/SALEOR-6516)

File: saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py [hunk 1]
```diff
@@ -6,42 +6,68 @@
 
 from .....channel.models import Channel
 from .....giftcard.models import GiftCard
+from .....graphql.tests.queries import subscription_queries
```
Comment (maarcingebala) [inline saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py:9]: `subscription_queries.py` could be moved to `tests/subscription_webhooks` I guess? I think it won't be needed anywhere else in Saleor.

File: saleor/graphql/tests/queries/fragments.py [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+          ...Price
+        }
+    }
+  }
+}
+"""
+)
+
+
+ADDRESS_DETAILS = """
```
Comment (maarcingebala) [inline saleor/graphql/tests/queries/fragments.py:213]: We already have ADDRESS, do we need this one as well?

File: saleor/plugins/webhook/tests/subscription_webhooks/payloads.py [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+            "name": menu_item.name,
+            "menu": {"id": graphene.Node.to_global_id("Menu", menu_item.menu_id)}
+            if menu_item.menu_id
+            else None,
+            "page": {"id": graphene.Node.to_global_id("Page", menu_item.page_id)}
+            if menu_item.page_id
+            else None,
+        },
+        "meta": None,
+    }
```
Comment (L3str4nge) [inline saleor/plugins/webhook/tests/subscription_webhooks/payloads.py:296]:
```
menu = {"id": graphene.Node.to_global_id("Menu", menu_item.menu_id)} if menu_item.menu_id  else None
page = {"id": graphene.Node.to_global_id("Page", menu_item.page_id)} if menu_item.page_id else None
return {
        "menuItem": {
            "id": menu_item_global_id,
            "name": menu_item.name,
            "menu": menu,
            "page": page
            else None,
        },
        "meta": None,
    }
```

File: saleor/graphql/tests/queries/fragments.py [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+      channel
+      products(first: 10) {
+        edges {
+            node {
+                ...BasicProductFields
+            }
+        }
+      }
+    }
+    """
```
Comment (IKarbowiak) [inline saleor/graphql/tests/queries/fragments.py:280]: Sometimes you add indentation and sometimes not, it will be nicer to use one approach.

**Review by IKarbowiak — APPROVED** [review]

File: saleor/plugins/webhook/tests/subscription_webhooks/subscription_queries.py [hunk 5]
```diff
@@ -62,27 +62,27 @@
...
-          }
-          shippingZone{
-            id
-            name
-          }
-        }
-      }
-    }
-"""
+# SHIPPING_PRICE_CREATED = """
```
Comment (L3str4nge) [inline saleor/plugins/webhook/tests/subscription_webhooks/subscription_queries.py:65]: Wy not removed?

Comment (jakubkuc) [reply saleor/plugins/webhook/tests/subscription_webhooks/subscription_queries.py:65]: by mistake, will fix

File: saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py (hunk 1 above)
Comment (jakubkuc) [reply saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py]: moved

File: saleor/plugins/webhook/tests/subscription_webhooks/payloads.py (hunk 3 above)
Comment (jakubkuc) [reply saleor/plugins/webhook/tests/subscription_webhooks/payloads.py:311]: done.

File: saleor/graphql/tests/queries/fragments.py (hunk 4 above)
Comment (jakubkuc) [reply saleor/graphql/tests/queries/fragments.py]: Fixed to match how it looks in playground after "prettyfi"

**Pull request merged**